api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -80,6 +80,8 @@ type VirtualMachineClassStatus struct {
 // +kubebuilder:printcolumn:name="CPU",type="string",JSONPath=".spec.hardware.cpus"
 // +kubebuilder:printcolumn:name="Memory",type="string",JSONPath=".spec.hardware.memory"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:printcolumn:name="VGPUDevices",type="string",priority=1,JSONPath=".spec.hardware.devices.vgpuDevices"
+// +kubebuilder:printcolumn:name="PassthroughDevices",type="string",priority=1,JSONPath=".spec.hardware.devices.dynamicDirectPathIODevices"
```

Comment:
This will spill out quite a bit of json/field names... thoughts on a simplified output with the same info: 

`{range .spec.hardware.devices.dynamicDirectPathIODevices[*]}{.vendorID}{':'}{.deviceID}{':'}{. customLabel}{','}{end}`

Reply:
Since the above expression will only list the values for `vendorID`, `deviceID` and `customLabel`, I don't think it will be apparent to the user what value corresponds to what field. I think having the key:value pair here will provide better context to the user.
What are your thoughts?

Reply:
The purpose of the get columns is for a quick glance, and just having the ids should be sufficient for that purpose. Having the entire json blobs be displayed will just wrap the lines and defeat the purpose. The user can always do a `kubectl describe` if they want to get more details.

Reply:
In that case, would it make sense to just display the device ID here?
Having `:` separated fields will not provide any useful information to the user IMO

Reply:
Cool with that